config.json

```diff
@@ -1172,6 +1172,15 @@
         "Recursion",
         "Searching"
       ]
+    },
+    {
```

Comment:
> The ordering of the core exercises is important.

We're actually reconsidering that. I need to open a discussion about it, but you can see the history over here: https://github.com/exercism/configlet/pull/34#issuecomment-320459371